### PR TITLE
add a new bootscreen theme

### PR DESCRIPTION
**Pull request by ammarbinfaisal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)
- [x] Added breif description with screenshots.

#### Changes proposed in this pull request:

bootscreen

<a href="https://user-images.githubusercontent.com/30975614/48977669-c1c57680-f0c4-11e8-9a42-85cd4237beb6.png" target="_blank">view it in properly</a>
Thank you!

### CODE REVIEW

File: meilix-artwork/debian/plymouth-theme-meilix-logo.install [hunk 1]
```diff
@@ -1 +1,2 @@
 debian/tmp/usr/share/plymouth/themes/meilix-logo/
+debian/tmp/usr/share/plymouth/themes/meilix-logo2/
```
Comment (pranav1698) [inline meilix-artwork/debian/plymouth-theme-meilix-logo.install]: Add a new line here for proper indentation

Comment (pranav1698) [reply meilix-artwork/debian/plymouth-theme-meilix-logo.install]: LGTM @ammarbinfaisal! Great work

**Comment by tabesin** [comment]
Why does your pr include changes to meilix-logo? Meilix-logo2 should be completely self-sustained, no?

**Comment by ammarbinfaisal** [comment]
So, should there be a different deb package named
plymouth-theme-melix-logo2 ?

**Comment by ammarbinfaisal** [comment]
#### made a separate package. any mistakes there?

bootscreen
I made circles also use the color meilix logo's wavy circular lines are using - the light red one. is this better? or should I use te dark one only.

**Comment by ammarbinfaisal** [comment]
**i think dark circles looks better, so i have made them dark again**

  
bootscreen2

**Comment by ammarbinfaisal** [comment]
@pranav1698 done

**Comment by ammarbinfaisal** [comment]
dnt know, it was generated somehow.
should I remove it?

File: meilix-artwork/debian/control [hunk 2]
```diff
@@ -12,6 +12,12 @@ Depends: ${misc:Depends}, plymouth, plymouth-label
 Description: graphical boot animation and logger - meilix-logo theme
  This package contains the default meilix-logo theme.
 
+Package: plymouth-theme-meilix-logo2
```
Comment (pranav1698) [inline meilix-artwork/debian/control]: Adding a hyphen before 2 will make it more clear!!
Also you can edit the description slightly, 
All the other things looks ready to ship!!

**Comment by tabesin** [comment]
You could also rename logo2 to plymouth-theme-meilix-ammar
;)

The idea is that we have three themes in the end. And the ammar theme package would be the default.

Let's me once more check it. I think however it could be merged in the current stage and then further improved. An issue like this should not stay a pending PR for too long...

**Pull request merged**